gupb_queue/models.py

```diff
@@ -1,13 +1,14 @@
 from django.db import models
 
 from tournament.models import Tournament
-
+from platforms.models import Platform
 
 class Queue(models.Model):
     name = models.CharField(max_length=30, blank=False)
     tournament = models.ForeignKey(Tournament, on_delete=models.CASCADE)
     start_date = models.DateTimeField()
     end_date = models.DateTimeField()
+    platform = models.ForeignKey(Platform, on_delete=models.SET_NULL, null=True)
```

Comment:
Ja bym może dał w tym miejscu `on_delete=models.CASCADE` i postawił `null=False,` bo i tak nie będą te platformy raczej często usuwane, plus to jest chyba rozwiązanie do błędu którego wskaże niżej

Reply:
null=False -> wyrzucić ten parametr